### ISSUE TITLE
Crash when spawning peds on the server with createped using artifact 5790 and newer

### ISSUE THREAD

**Issue by gewrutv** [citizenfx/fivem#1565]
[pedtest.zip](https://github.com/citizenfx/fivem/files/9299746/pedtest.zip)
[CfxCrashDump_2022_08_10_11_50_36.zip](https://github.com/citizenfx/fivem/files/9299747/CfxCrashDump_2022_08_10_11_50_36.zip)

I have discovered an issue that exist in artifact 5790 and beyond with spawning peds on the server. It appears whenever I spawn peds on the server that will be owned by another client it will then crash the client or even potentially both clients. The above resource, pedtest, is the resource I created to reproduce this issues. I have also added links to videos showing the testing. Spawning the peds on the client has no issues at all.

pedtest - artifact 5789 - https://streamable.com/lhj3yu

pedtest - artifact 5799 - https://streamable.com/q0kl25

The attached files are the resource and the crash dump from the client.

**Comment by blattersturm** [comment]
Thanks for the repro - someone had previously reported something regarding this but didn't have a repro so it wasn't really able to be looked into (don't really have much time for random wild goose chases).

This'll be looked into whenever possible.

**Comment by blattersturm** [comment]
OK, this is a bit weird:

1. You didn't specify _how to use the repro_ (apparently have to set coords to `crun SetEntityCoords(PlayerPedId(), vector3(2379.44, 5060.68, 46.44))` and then run `pedtestserver` somewhere).
2. I'm probably doing something else wrong, as my game remains functioning _fine_ when using `pedtestserver` on current code, even when flying back and forth and ownership transfer is definitely happening.

**Comment by gewrutv** [comment]
> OK, this is a bit weird:
> 
> 1. You didn't specify _how to use the repro_ (apparently have to set coords to `crun SetEntityCoords(PlayerPedId(), vector3(2379.44, 5060.68, 46.44))` and then run `pedtestserver` somewhere).
> 2. I'm probably doing something else wrong, as my game remains functioning _fine_ when using `pedtestserver` on current client/server code (although using build 2699), even when flying back and forth and ownership transfer is definitely happening.

Yes, sorry I should have specificed to TP to those coords. This is part of our chicken catching script so I was spawning the peds at a specific spot. It only seemed to happen when spawning the peds tried to transfer ownership directly after the peds had been spawned. 
[CfxCrashDump_2022_08_10_18_34_20.zip](https://github.com/citizenfx/fivem/files/9303039/CfxCrashDump_2022_08_10_18_34_20.zip)

I was able to duplicate the crash on 2699 as well and have include new crash logs

If you want to save yourself some trouble you can use
`RegisterCommand('tptotest', function()
    SetEntityCoords(PlayerPedId(), chickenSpawns[1])
end)`

This is on artifact 5799 with build 2699

**Comment by gewrutv** [comment]
[CfxCrashDump_2022_08_10_19_01_15.zip](https://github.com/citizenfx/fivem/files/9303368/CfxCrashDump_2022_08_10_19_01_15.zip)

I wanted to get someone else to help test instead of using cl-2 we are able to reproduce the results on 2699 two completely independent clients

**Comment by blattersturm** [comment]
I'm *still* not able to replicate this issue. I suspected this to be a retail/debug build difference, but even with retail client/server (5800, current one) I'm not able to get the game to crash at all around this area with the server-side chickens.

**Comment by blattersturm** [comment]
Mildly curious, however, is that I can't reproduce this change on my local server working tree which had some _other_ fixes applied regarding 'first-frame updates'. It could be something regarding server-side rejection of script disowning actually _breaking_ something without the first-frame update fix, but relying on that interpretation is a little risky.

**Comment by AvarianKnight** [comment]
That commit seems to have fixed

**Comment by blattersturm** [comment]
GitHub, please. The commit said 'didn't fix #1565', not 'fixes #1565'. 🤦🏻

**Comment by AngelofDeathGaming** [comment]
Can we get an update or an ETA of when this will be resolved?
